ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  mig-operator:
```

Comment:
I don't think we need a base of mig-operator, we may need a base of ansible-operator though